ramalama

```diff
@@ -342,6 +342,11 @@ def main(args):
     conman = select_container_manager()
     ramalama_store = get_ramalama_store()
 
+    if conman:
```

Comment:
Ok so we default to running a container for use with the server.

Reply:
Sounds good to me, the other problem is we need to add:

pip install "huggingface_hub[cli]==0.24.2"

to the install script but that's no biggie.

Reply:
Re-pushed, it now does the re-exec in a container thing for just run/serve and installs the huggingface dependency in install script